_translations/fr.yml

```diff
@@ -121,6 +121,7 @@ fr:
     walletninki: "Ninki est un portefeuille multi-signature avec une interface utilisateur magnifique qui rend l'envoi de bitcoins aussi facile qu'un email. Vous avez le contrôle total de vos bitcoins en tout temps."
     walleteclairmobile: "Eclair Mobile est un portefeuille Bitcoin de nouvelle génération, prêt pour Lightning. Il peut s'utiliser comme un portefeuille Bitcoin standard, et peut aussi se connecter au réseau Lightning, pour des paiements rapides et instantanés."
     walletwasabi: "Wasabi est un portefeuille Bitcoin protégeant votre vie privée."
+    walletunstoppable: "Unstoppable est un portefeuille crypté sécurisé et entièrement décentralisé prenant en charge des principales crypto-monnaies, notamment les jetons Bitcoin."
```

Comment:
Please ensure that the description is consistent with the English description.

Reply:
done